pylmod/gradebook.py

```diff
@@ -76,6 +76,25 @@ def unravel_sections(section_data):
                 sections.append(section)
         return sections
 
+    @staticmethod
+    def unravel_staff(staff_data):
```

Comment:
It isn't private because we don't unravel by default.  Without `simple=True` we return the structure as returned by the service and want to provide a way to unravel that data and keep the full "non-simple" structure to consumers of the class.